man/ri.1

```diff
@@ -35,6 +35,8 @@ can be:
 .Pp
 .It Class::method | Class#method | Class.method | method
 .Pp
+.It ruby: | ruby:packed_data | ruby:dig_methods
```

Comment:
Maybe we can make this example align with ri usage message
https://github.com/ruby/rdoc/blob/2824bb6910e7606bd2d6a849c356a55d2cfd2dcd/lib/rdoc/ri/driver.rb#L122-L130